kale/embed/__init__.py

```diff
@@ -0,0 +1,42 @@
+# from .attention_cnn import CNNTransformer, ContextCNNGeneric
```

Comment:
Confused and hard to review this change? Why is this is commented? Do you need a review on this or not? If merged (you said it is ready), the users will be confused as well. At least add some docstrings at the top to explain so that not only you know what's going on here.

Reply:
A guess this is a problem of documentation. Please pay attention.